code/frontend/src/actions/contractor-info-actions.js

```diff
@@ -81,7 +81,7 @@ function addEngagementContract(projectData, contractorId) {
        project["resourceId"] = "";
        project = conformDropdownValuesToDefault(project);
        let req = request
-        .post(`${LIVE_SITE}contractors/add/engagementContract`)
+        .post(`${LOCALHOST}contractors/add/engagementContract`)
```

Comment:
Shouldn't this be LIVE_SITE?

Reply:
will deploy back-end and change this to LIVE_SITE